thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/client/thrift/THttpClientDelegate.java

```diff
@@ -201,6 +201,26 @@ public RpcResponse execute(ClientRequestContext ctx, RpcRequest call) {
         return reply;
     }
 
+    private static String getAuthority(ClientRequestContext ctx) {
+        String authority = ctx.additionalRequestHeaders().get(HttpHeaderNames.AUTHORITY);
```

Comment:
ditto Let's use early return and assign to a local variable to reuse it.

Reply:
Commented on https://github.com/line/armeria/pull/4520#discussion_r1040597683